src/EnergyPlus/WeatherManager.hh

```diff
@@ -911,169 +914,29 @@ struct WeatherManagerData : BaseGlobalStruct
     Weather::AnnualMonthlyDryBulbWeatherData OADryBulbAverage; // processes outside air drybulb temperature
 
     // SetCurrentWeather static vars
-    int NextHour;
+    int NextHour = 1;
 
     // ReadEPlusWeatherForDay static vars
-    int CurDayOfWeek;
-    Real64 ReadEPlusWeatherCurTime;
-    bool LastHourSet;
+    int CurDayOfWeek = 1;
+    Real64 ReadEPlusWeatherCurTime = 1.0;
+    bool LastHourSet = false;
 
     Weather::WeatherVars wvarsLastHr;
     Weather::WeatherVars wvarsNextHr;
 
-    Real64 IsRainThreshold; // precipitation threshold (m) for a rainy day
+    Real64 IsRainThreshold = 0.8; // precipitation threshold (mm) for timestep IsRain (divided later by NumOfTimeStepInHour)
 
     // ProcessEPWHeader static vars
-    std::string EPWHeaderTitle;
+    std::string EPWHeaderTitle = "";
 
     void clear_state() override
     {
-        this->YearOfSim = 1;             // The Present year of Simulation.
-        this->EnvironmentReportNbr = 0;  // Report number for the environment stamp
-        this->EnvironmentReportChr = ""; // Report number for the environment stamp (character -- for printing)
-        this->WeatherFileExists = false; // Set to true if a weather file exists
-        this->LocationTitle = "";        // Location Title from input File
-        this->LocationGathered = false;  // flag to show if Location exists on Input File (we assume one is
-
-        this->GetBranchInputOneTimeFlag = true;
-        this->GetEnvironmentFirstCall = true;
-        this->PrntEnvHeaders = true;
-        this->WeatherFileLatitude = 0.0;
-        this->WeatherFileLongitude = 0.0;
-        this->WeatherFileTimeZone = 0.0;
-        this->WeatherFileElevation = 0.0;
         this->siteShallowGroundTempsPtr.reset();
         this->siteBuildingSurfaceGroundTempsPtr.reset();
         this->siteFCFactorMethodGroundTempsPtr.reset();
         this->siteDeepGroundTempsPtr.reset();
```

Comment:
But kept these in `clear_state` before the `new` because I wasn't sure about these.

Reply:
Yep, I support this temporarily.  These need to be changed, and I already know what to do, just need to do it.